server/src/db/migrations/20240429191843_createSandwichTable.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+	return knex.schema.createTable("sandwiches", (table) => {
+		table.bigIncrements("id")
+		table.string("name").notNullable().unique()
+		table.string("imgUrl")
+		table.string("restaurant").notNullable()
+		table.string("description")
+		table.timestamp("createdAt").notNullable().defaultTo(knex.fn.now())
+		table.timestamp("updatedAt").notNullable().defaultTo(knex.fn.now())
+	})
+};
```

Comment:
this may not take affect until a later PR, but do please make sure everyone updates their tab size to 2
https://ior.ad/9eO8?iframeHash=viewsteps